app/cdap/components/ConnectionList/index.tsx

```diff
@@ -86,18 +98,17 @@ export default function ConnectionList() {
       return {
         ...eachConnectorType,
         count: connections.length,
-        icon: <GCSIcon />,
+        icon: eachConnectorType.SVG,
       };
     });
 
-    const firstLevelData = connectorTypes.filter((each) => {
-      if (each.count > 0) {
-        return {
-          name: each.name,
-          count: each.count,
-        };
+    // Connector types which has connections inside it
+    const firstLevelData = connectorTypes.filter((eachConnectorType) => {
```

Comment:
always check whether a variable is not null and array and has length when u want to use array methods

Reply:
resolved

Reply:
resolved